### PR TITLE
Remove feature descriptions in `FEATURES.md`

### PR DESCRIPTION
**Pull request by madsmtm**
Builds upon https://github.com/rust-windowing/winit/pull/3478. Related to https://github.com/rust-windowing/winit/issues/3431.

I went through all of the text in `FEATURES.md`, and cross-referenced it with our code, and as I suspected, the information in the code is much more detailed and up to date. So I propose we remove most of the contents in `FEATURES.md`, as it is difficult to keep up to date, and I don't really see the value provided.

Only thing I'm unsure about is the following three points that are yet unimplemented, and as such doesn't have a clear reference in our code:

> - **Popup / modal windows**: Windows can be created relative to the client area of other windows, and parent windows can be disabled in favor of popup windows. This feature also guarantees that popup windows get drawn above their owner.
> - **Gamepad/Joystick events**: Capturing input from gamepads and joysticks.
> - **Device movement events**: Capturing input from the device gyroscope and accelerometer.

### CODE REVIEW

**Review by daxpedda — COMMENTED** [review]
I think this is the right way to go.
Will take a closer look when its ready.

**Comment by kchibisov** [comment]
bump

File: FEATURES.md
Comment (daxpedda) [inline FEATURES.md:1]: I feel like this file is now quite small and its contents should either live in `README.md`, `CONTRIBUTING.md` or be removed entirely.

**Review by daxpedda — APPROVED** [review]
LGTM.

My small nit can be addressed in a follow-up.

**Comment by daxpedda** [comment]
@madsmtm do you still want to move on with this?

**Review by notgull — APPROVED** [review]

**Pull request merged**